### ISSUE TITLE
build issue

### ISSUE THREAD

**Issue by felixlinker** [EDCD/coriolis#340]
Commit 0ac44ac2678198a483a82f697398ecec98e2695f doesn't let me start the server locally. If checking out the previous one, everything works fine.

This is the log:
```
> coriolis_shipyard@2.9.17 start C:\Users\felix\git\edcd\coriolis
> node devServer.js

Listening at localhost:3300
C:\Users\felix\git\edcd\coriolis\build
Time: 12440ms
                                 Asset      Size  Chunks                    Chunk Names
                                app.js   5.23 MB       0  [emitted]  [big]  app
  602599e5522bc6834a13694904748991.eot   19.1 kB          [emitted]
73ff8a514a9a68d88a694f6c543f15a2.woff2   17.1 kB          [emitted]
  cf9b6351a340f44577fff4a61748b714.ttf   40.8 kB          [emitted]
  ed284abd139ee9ff622bbd89a8fbd09c.svg   49.3 kB          [emitted]
  50eb5f8ff63b2bfe8903441048684c85.svg   30.6 kB          [emitted]
 8d9d96f2121135b1cd272fa1f7eed470.woff   22.7 kB          [emitted]
                                lib.js   1.25 MB       1  [emitted]  [big]  lib
                               app.css   50.4 kB       0  [emitted]         app
                            app.js.map   6.15 MB       0  [emitted]         app
                           app.css.map  84 bytes       0  [emitted]         app
                            lib.js.map    1.5 MB       1  [emitted]         lib
                            index.html   1.03 kB          [emitted]

ERROR in Template execution failed: TypeError: Cannot read property 'toISOString' of
undefined

ERROR in   TypeError: Cannot read property 'toISOString' of undefined

  - index.ejs:17803
    C:/Users/felix/git/edcd/coriolis/src/index.ejs:17803:37

  - index.ejs:17818 module.exports
    C:/Users/felix/git/edcd/coriolis/src/index.ejs:17818:3

  - index.js:264
    [coriolis]/[html-webpack-plugin]/index.js:264:16

  - util.js:16 tryCatcher
    [coriolis]/[bluebird]/js/release/util.js:16:23

  - promise.js:512 Promise._settlePromiseFromHandler
    [coriolis]/[bluebird]/js/release/promise.js:512:31

  - promise.js:569 Promise._settlePromise
    [coriolis]/[bluebird]/js/release/promise.js:569:18

  - promise.js:606 Promise._settlePromiseCtx
    [coriolis]/[bluebird]/js/release/promise.js:606:10

  - async.js:138 Async._drainQueue
    [coriolis]/[bluebird]/js/release/async.js:138:12

  - async.js:143 Async._drainQueues
    [coriolis]/[bluebird]/js/release/async.js:143:10

  - async.js:17 Immediate.Async.drainQueues
    [coriolis]/[bluebird]/js/release/async.js:17:14


Child html-webpack-plugin for "index.html":
         Asset    Size  Chunks  Chunk Names
    index.html  571 kB       0
Child extract-text-webpack-plugin:
                                     Asset     Size  Chunks             Chunk Names
      602599e5522bc6834a13694904748991.eot  19.1 kB          [emitted]
     8d9d96f2121135b1cd272fa1f7eed470.woff  22.7 kB          [emitted]
    73ff8a514a9a68d88a694f6c543f15a2.woff2  17.1 kB          [emitted]
      cf9b6351a340f44577fff4a61748b714.ttf  40.8 kB          [emitted]
      ed284abd139ee9ff622bbd89a8fbd09c.svg  49.3 kB          [emitted]
      50eb5f8ff63b2bfe8903441048684c85.svg  30.6 kB          [emitted]
webpack: Failed to compile.
```

**Comment by felixlinker** [comment]
Problem seems to be line 29-30 in `index.ejs`: https://github.com/EDCD/coriolis/commit/0ac44ac2678198a483a82f697398ecec98e2695f#diff-c6429b99ea286602fe43ed65b7efad81R29.

Changing those to what follows lets me build succesfully, i.e. `htmlWebpackPlugin.options.date` is undefined. However I have no clue, whether that is a good change.
```js
window.CORIOLIS_DATE = '<%- new Date().toISOString().slice(0, 10)  %>';
window.BUGSNAG_VERSION = '<%- htmlWebpackPlugin.options.version + '-' + new Date().toISOString()  %>';
```

**Comment by willyb321** [comment]
Fixed in 0eb9a2048d5340b622d234ee0451949fabcbe3f3
